docker/CMSRucioClient/scripts/account_to_scope_mapping.py

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python3
+
+import argparse
+from rucio.client import Client
+from rucio.common.exception import Duplicate
+
+def add_group_account_scopes(rclient, include_local_users_accounts, only_include_accounts_with_group_suffix, dry_run=True):    
+    group_accounts = rclient.list_accounts(account_type="GROUP")
+    group_scopes = [ scope for scope in rclient.list_scopes() if scope.startswith("group.")]
+
+    desired_group_scopes = set()
+    existing_group_scopes = set()
+
+    for account in group_accounts:
+        account_name = account["account"]
+        scope_name = f"group.{account_name}"
+        if account_name.endswith("_local_users") or account_name.endswith("_local"):
+            if include_local_users_accounts:
+                desired_group_scopes.add((account_name, scope_name))
+        elif only_include_accounts_with_group_suffix:
+            if account_name.endswith("_group"):
+                desired_group_scopes.add((account_name, scope_name))
+        else:
+            desired_group_scopes.add((account_name, scope_name))  
+
+    
+    group_prefix_len = len("group.")
+    for scope in group_scopes:
+        account = scope[group_prefix_len:]
+        existing_group_scopes.add((account, scope))
+
+    
+    scopes_to_add = desired_group_scopes - existing_group_scopes
+
+    if dry_run:
+        for account, scope in scopes_to_add:
+            print(account, scope) 
```

Comment:
Make this more like line 42, where you say what would be done

Reply:
`Dry run` is unrelated to Line 42. 
- when `--dry-run` is `True` - we just print the names of scopes that will be created instead of actually creating them.

The comment on 42 is because in 33 we already remove the existing scopes, so no question of duplication.
https://github.com/dmwm/CMSRucio/blob/a060b8870968a2b935b51139ccb791acc6c71d53/docker/CMSRucioClient/scripts/account_to_scope_mapping.py#L33

But I kept it for safety.

Reply:
I only mean that it should be a readable message with text rather than just a couple of variables printed out.

Reply:
Ah Sorry, I misunderstood. I looked at it after removing the extra space :sweat_smile: , so related it to `except Duplicate`. 
Will do the change asap.

Reply:
Ah Sorry, I misunderstood. I looked at it after removing the extra space :sweat_smile: , so I related it to `except Duplicate`. 
Will do the change asap.